osquery/config/config.cpp

```diff
@@ -530,6 +541,54 @@ void stripConfigComments(std::string& json) {
   json = sink;
 }
 
+std::map<std::string, std::string> Config::restoreConfigBackup() {
+  if (!FLAGS_config_enable_backup) {
```

Comment:
Just a question, why do you need to switch config persistence off?

Reply:
Just because people might not want to use it. If config is provided using reliable plugin, for example filesystem, it makes less sense to back up the config.